src/util/format.ts

```diff
@@ -202,19 +202,26 @@ interface GetTooltipMarkerOpt {
     // id name for marker. If only one marker is in a rich text, this can be omitted.
     // By default: 'markerX'
     markerId?: string;
+    size?: string;
 }
 // Only support color string
-export function getTooltipMarker(color: ColorString, extraCssText?: string): TooltipMarker;
+export function getTooltipMarker(color: ColorString, size?: string, extraCssText?: string): TooltipMarker;
 export function getTooltipMarker(opt: GetTooltipMarkerOpt): TooltipMarker;
 export function getTooltipMarker(inOpt: ColorString | GetTooltipMarkerOpt, extraCssText?: string): TooltipMarker {
     const opt = zrUtil.isString(inOpt) ? {
         color: inOpt,
-        extraCssText: extraCssText
+        extraCssText: extraCssText,
+        size: inOpt
```

Comment:
`getTooltipMarker ` was called in [here](https://github.com/apache/incubator-echarts/blob/next/src/component/tooltip/tooltipMarkup.ts#L484)
So you might want add an attribute `size` for  `getTooltipMarker`
```js
const marker = getTooltipMarker({
    color: colorStr,
    type: markerType,
    renderMode,
    markerId: markerId,
    size: size
});
```
You should track all the way up to find the function that starts to build marker and pass `size` at that function

Reply:
ok.